bot/cogs/watchchannels/talentpool.py

```diff
@@ -224,7 +224,7 @@ def _nomination_to_string(self, nomination_object: dict) -> str:
                 Status: **Active**
                 Date: {start_date}
                 Actor: {actor.mention if actor else actor_id}
-                Reason: {nomination_object["reason"]}
+                Reason: {textwrap.shorten(nomination_object["reason"], width=200, placeholder="...")}
```

Comment:
Did you confirm that these need to be shortened and that the paginator doesn't help?

Reply:
Yes, because this takes it as one line.